Module not found error on svelte-loader/webpack

Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `bits-ui@2.8.13` for the project I'm working on.

<!-- 🔺️🔺️🔺️ PLEASE REPLACE THIS BLOCK with a description of your problem, and any other relevant context 🔺️🔺️🔺️ -->

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/bits-ui/dist/bits/menubar/menubar.svelte.js b/node_modules/bits-ui/dist/bits/menubar/menubar.svelte.js
index 039538f..36dbf18 100644
--- a/node_modules/bits-ui/dist/bits/menubar/menubar.svelte.js
+++ b/node_modules/bits-ui/dist/bits/menubar/menubar.svelte.js
@@ -4,7 +4,7 @@ import { createBitsAttrs, getAriaExpanded, getDataDisabled, getDataOpenClosed, }
 import { kbd } from "../../internal/kbd.js";
 import { wrapArray } from "../../internal/arrays.js";
 import { onMount } from "svelte";
-import { getFloatingContentCSSVars } from "../../internal/floating-svelte/floating-utils.svelte";
+import { getFloatingContentCSSVars } from "../../internal/floating-svelte/floating-utils.svelte.js";
 import { RovingFocusGroup } from "../../internal/roving-focus-group.js";
 const menubarAttrs = createBitsAttrs({
     component: "menubar",
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>


```
ERROR in ../../node_modules/bits-ui/dist/bits/menubar/menubar.svelte.js 16:0-97
Module not found: Error: Can't resolve '../../internal/floating-svelte/floating-utils.svelte' in '/Users/xxx/workspace/xxx/node_modules/bits-ui/dist/bits/menubar'
Did you mean 'floating-utils.svelte.js'?
BREAKING CHANGE: The request '../../internal/floating-svelte/floating-utils.svelte' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
```

--- a/.changeset/bitter-brooms-thank.md
+++ b/.changeset/bitter-brooms-thank.md
@@ -1,0 +1,5 @@
+---
+"bits-ui": patch
+---
+
+fix(Menubar): ensure extension exists in import

--- a/packages/bits-ui/src/lib/bits/menubar/menubar.svelte.ts
+++ b/packages/bits-ui/src/lib/bits/menubar/menubar.svelte.ts
@@ -20,7 +20,7 @@ import { wrapArray } from "$lib/internal/arrays.js";
 import type { OnChangeFn, RefAttachment, WithRefOpts } from "$lib/internal/types.js";
 import { onMount } from "svelte";
 import type { FocusEventHandler, KeyboardEventHandler, PointerEventHandler } from "svelte/elements";
-import { getFloatingContentCSSVars } from "../../internal/floating-svelte/floating-utils.svelte";
+import { getFloatingContentCSSVars } from "../../internal/floating-svelte/floating-utils.svelte.js";
 import { RovingFocusGroup } from "$lib/internal/roving-focus-group.js";
 
 const menubarAttrs = createBitsAttrs({

